src/TextField/TextField.jsx

```diff
@@ -70,7 +71,9 @@ export const TextField = ({
           {label}
         </FormLabel>
       )}
-      <InputBase {...other} id={id} />
+
+      <InputBase {...other} id={id} {...InputProps} />
```

Comment:
Hmmm, since we already have `{...other}` spread for the `InputBase`, we can avoid having to declare `{...InputProps}` too.

You can merge the `InputProps` object with the `other` at the `AutocompleteTextField.renderInput` method using the merge spread:

```js
function renderInput ({ InputProps, classes, fullWidth, ref, ...other }) {
  const otherProps = { ...other, ...InputProps }

  return (
    <TextField
      fullWidth
      inputRef={ref}
      {...otherProps}
    />
  )
}
```

With this, you can leave the `TextField` exactly the same as `master`.

Reply:
Done!